### PR TITLE
Set Xray version 2.3.2

### PR DESCRIPTION
**Pull request by eldada**
Set Xray version 2.3.2

### CODE REVIEW

**Comment by jainishshah17** [comment]
@eldada As we discussed please bumpup rabbitmq-ha version in `requirements.yaml`

**Comment by jainishshah17** [comment]
@eldada Now we don't need `ci/test-values.yaml`. Can you please remove it.

File: stable/xray/values.yaml [hunk 1]
```diff
@@ -111,12 +111,22 @@ mongodb:
 ##
 rabbitmq-ha:
   replicaCount: 1
-  rabbitmqUsername: "guest"
```
Comment (ankushchadha) [inline stable/xray/values.yaml]: I think the quotes were added for a reason, maybe this is not applicable to the new chart.

Comment (eldada) [reply stable/xray/values.yaml]: Yes. Was already tested with `guest` and works well.

**Pull request merged**